x/posts/legacy/v0.1.0/types.go

```diff
@@ -15,16 +15,16 @@ type GenesisState struct {
 // PostID represents a unique post id
 type PostID uint64
 
-// Post is a struct of a post
+// TextPost is a struct of a post
 type Post struct {
 	PostID            PostID         `json:"id"`                 // Unique id
-	ParentID          PostID         `json:"parent_id"`          // Post of which this one is a comment
+	ParentID          PostID         `json:"parent_id"`          // TextPost of which this one is a comment
```

Comment:
Refactoring error, I think this should be `Post` instead

```suggestion
	ParentID          PostID         `json:"parent_id"`          // Post of which this one is a comment
```